### ISSUE TITLE
poetry init -n prints stuff irrelevant to no-interaction mode

### ISSUE THREAD

**Issue by geckon** [python-poetry/poetry#2088]
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: openSUSE Tumbleweed (up-to-date)
- **Poetry version**: 1.0.3
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: reproducible without pyproject.toml 

## Issue

Hello!

This is a minor issue but `poetry init` with `-n|--no-interaction` prints output that is IMO irrelevant to no-interaction mode:

```
$ poetry init --no-interaction

This command will guide you through creating your pyproject.toml config.


You can specify a package in the following forms:
  - A single name (requests)
  - A name and a constraint (requests ^2.23.0)
  - A git url (git+https://github.com/python-poetry/poetry.git)
  - A git url with a revision (git+https://github.com/python-poetry/poetry.git#develop)
  - A file path (../my-package/my-package.whl)
  - A directory (../my-package/)
  - An url (https://example.com/packages/my-package-0.1.0.tar.gz)
```

Both parts of the output only really make sense in an interactive mode.

Please note that there is a work-around (`poetry init -n -q`) but that will suppress all output which is not what you always want.

For example consider running `init` with pyproject.toml already existing:
```
$ poetry init -n -q
$ poetry init -n 
A pyproject.toml file already exists.
```

Thanks!

**Comment by nikhilweee** [comment]
I second this! The output of `poetry init -n` is misguiding.

**Comment by finswimmer** [comment]
This is solved in in the current preview release (1.2.0b1).
